### DISCUSSION TITLE
How can I run taiko execution on remote windows clients?

### DISCUSSION THREAD

**Discussion by ShayWeizmanFG** [getgauge/taiko#2591] [General]
Is it possible to run taiko execution on remote windows client (Like selenium grid)? Should I install taiko on the remote windows client??

**Comment by saikrishna321** [comment]
Please see taiko-selenoid plug-in 

https://github.com/saikrishna321/taiko-selenoid

**Reply by ShayWeizmanFG** [reply]
Hi Saikrishna,

Can you explain a bit about how it works?

The selenoid should be install on the windows client (Where the tests are running?)?

Thanks!!

**Comment by ShayWeizmanFG** [comment]
Hi Saikrishna,

I have succeeded to run taiko on windows using taiko-selenoid, but it is running it inside a docker on the remote windows machine.
I need to run taiko on the windows itself and not on a docker on windows. Is it possible?

Thanks!!
Shay
